packages/schematics/integration/integration.spec.ts

```diff
@@ -0,0 +1,260 @@
+import {
```

Comment:
I really like the structure of these tests.
I can imagine moving the commands and the expected output to a JSON-file and executing all tests with a single `it.each`. ❤️